server/controllers/artist.js

```diff
@@ -22,7 +24,18 @@ router.get('/:artistId', (req, res) => {
   .catch(err => {
     res.status(400).send('Get artist info error!');
   });
+});
 
+router.post('/profile', authenticate, (req, res) => {
+  const profile = req.body.profile;
+  const artistId = req.user.userId;
+  model.profile(artistId, profile)
```

Comment:
modle.profile? Don't see a function called profile in queries.js. Sure this work?